hs_collection_resource/views.py

```diff
@@ -104,3 +111,151 @@ def update_collection_for_deleted_resources(request, shortkey, *args, **kwargs):
     finally:
         return JsonResponse(ajax_response_data)
 
+
+def update_collection_coverages(request, shortkey, *args, **kwargs):
```

Comment:
It is more important to test public functions. If the url for this view function somehow got changed, there would be no way to detect it until user reports the problem.